NEWS

```diff
@@ -56,6 +57,8 @@
 - Fixes in memcpy selection for AMD
 - Fixed in UCT/UD endpoint flush functionality
 - Fixes in XPMEM detection
+- Fixes in rendezvous staging protocol
+- Fixes in ROCE mlx5 UD port configuration
```

Comment:
Fixes in RoCEv1 UDP.src_port configuration with DevX

Reply:
is this UDP or UD ?  UDP.src_port  - too much details.

Reply:
it's UDP (layer4). RDMA headers (UD/RC/..) and after layer4 headers.

Reply:
kk